package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "postinstall": "npm run bootstrap",
     "bootstrap": "lerna bootstrap --ignore-scripts --include-dependencies --no-ci --hoist && npm run temporaryClientFixes && npm run build",
-    "temporaryClientFixes": "cp node_modules/interface-datastore/dist/src/key.d.ts node_modules/interface-datastore/src && cd node_modules/bcrypto && npm i",
+    "temporaryClientFixes": "rm -r node_modules/acorn && cp -r node_modules/karma-typescript/node_modules/acorn/ node_modules/acorn/ && cp node_modules/interface-datastore/dist/src/key.d.ts node_modules/interface-datastore/src && cd node_modules/bcrypto && npm i",
```

Comment:
Can you give a (very short) description what is happening here (or have I missed?)?

Reply:
yes I should have noted this, thanks.

this is a small temporary workaround fix for an updated `acorn` version which `karma-typescript` needs for the `es6-transform` plugin (which i described here: https://github.com/ethereumjs/ethereumjs-monorepo/pull/1177#discussion_r607922114). karma was hanging on the transform step when using an older version of `acorn`. our node_modules has several `acorn` version but the one on the top level that was being used was outdated, so i copy the newer version located inside `karma-typescript/node_modules` to the top level `node_modules` which resolved the problem.

i'll check back in a few weeks and if the issue is resolved (most likely by other modules updating their `acorn` dependency version) i can remove this temporary workaround 👍